### PR TITLE
[REF] product_customer_code: Migrate module product_customer_code to new api

### PR DESCRIPTION
**Pull request by ysantiago**

### CODE REVIEW

File: product_customer_code/__openerp__.py [hunk 1]
```diff
@@ -1,60 +1,25 @@
...
-##############################################################################
+# coding: utf-8
+# Copyright 2017 Vauxoo (https://www.vauxoo.com) info@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
 {
     "name": "Products Customer Code",
-    "version": "1.0",
-    "author": "Vauxoo,Odoo Community Association (OCA)",
+    "version": "8.0.0.0.1",
```
Comment (moylop260) [inline product_customer_code/__openerp__.py]: Use 8.0.1.0.0

File: product_customer_code/README.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Credits
+=======
+
+
+Contributors
+------------
+
+* Julio Serna Hernández <julio@vauxoo.com>
+* Yennifer Santiago <yennifer@vauxoo.com>
+* Moises Lopez <moylop260@vauxoo.com>
```
Comment (moylop260) [inline product_customer_code/README.rst]: Moisés López

**Review by JulioSerna — APPROVED** [review]
👍

**Review by moylop260 — APPROVED** [review]
:+1:
Red travis is unrelated since that stable branch has the same error result too

**Comment by moylop260** [comment]
@pedrobaeza @max3903 Could you validate the CLA to merge this one?

**Comment by AaronHForgeFlow** [comment]
It seems that `product_customer_code` is overlapped by `product_supplierinfo_for_customer`. The last one is merged in v8 and there was a previous Pull Request for product_customer_code in v8 (https://github.com/OCA/product-attribute/pull/122) that was closed for the same reason. IMHO I agree to maintain  `product_supplierinfo_for_customer` instead of `product_customer_code`

**Comment by moylop260** [comment]
@aheficent You are right!
Thanks for reply

Note: We should add a note in the readme about a overlapping module.

**Pull request closed without merging**